test/unit/functions/functions.spec.ts

```diff
@@ -216,14 +216,17 @@ test.each([
     `.expectToMatchJsResult();
 });
 
-test.each([tstl.LuaTarget.Lua51, tstl.LuaTarget.Universal])("function.length unsupported (%p)", luaTarget => {
-    util.testFunction`
+test.each([tstl.LuaTarget.Lua50, tstl.LuaTarget.Lua51, tstl.LuaTarget.Universal])(
+    "function.length unsupported (%p)",
+    luaTarget => {
+        util.testFunction`
         function fn() {}
         return fn.length;
     `
```

Comment:
indenting seems a bit wonky here, though that might be github struggling to show whitespace diffs. Normally we indent code in our test functions 1 more than the util.testFunction line they are for:
```suggestion
        util.testFunction`
            function fn() {}
            return fn.length;
        `
```